REQUIRE

```diff
@@ -1,3 +1,2 @@
-julia 0.6
-DataFrames 0.11
-Compat 0.57.0
```

Comment:
This PR takes out Compat, but  src/DataFramesMeta.jl still imports it.

Reply:
#107 fixes this